### PR TITLE
104 make site not crash when no machine is in database (Presentation)

### LINKED ISSUE #104: make site not crash when no machine is in database (Presentation)

### PR DESCRIPTION
**Pull request by Kaszz**
Making the site not crash when list is empty or when the connection between database and backend is failing, causing a response with status code other than 200.

close #104

### CODE REVIEW

File: src/test/java/com/brewmes/demo/BrewMESApplicationTests.java [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import org.springframework.boot.test.context.SpringBootTest;
 
 @SpringBootTest
-class BrewMESApplicationTests {
+class BrewMesApplicationTests {
 
 	@Test
 	void contextLoads() {
```
Comment (Lawst96) [inline src/test/java/com/brewmes/demo/BrewMESApplicationTests.java:10]: Why does this class even exist?

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #Datasource setup
 spring.datasource.url=jdbc:postgresql://localhost:5432/brewmes
 spring.datasource.username=postgres
-spring.datasource.password=#password 
+spring.datasource.password=0205
```
Comment (Lawst96) [inline src/main/resources/application.properties:4]: Might want to not commit your birthday

Comment (Wylardt) [reply src/main/resources/application.properties:4]: Yea, where's the 95?

File: src/test/java/com/brewmes/demo/BrewMESApplicationTests.java (hunk 1 above)
Comment (Kaszz) [reply src/test/java/com/brewmes/demo/BrewMESApplicationTests.java:10]: Its a default testclass made by SpringBoot. After the changing of names this class caused me troubles so i had to rename it. Not sure if we can just delete it.

File: src/main/resources/application.properties (hunk 2 above)
Comment (Teyson) [reply src/main/resources/application.properties:4]: I like you anyway <3

**Review by Teyson — APPROVED** [review]

File: src/main/resources/application.properties (hunk 2 above)
Comment (dottore1) [reply src/main/resources/application.properties:4]: it's everyones birthday,

**Review by Lawst96 — APPROVED** [review]

**Review by Wylardt — APPROVED** [review]

**Review by dottore1 — APPROVED** [review]

**Review by Lise4070 — APPROVED** [review]

**Pull request merged**